src/envoy/http/mixer/filter.h

```diff
@@ -104,6 +103,11 @@ class Filter : public StreamFilter,
   // trailers.
   uint64_t request_total_size_{0};
 
+  // True for gRPC requests
```

Comment:
Nit: add dot at the end of your comment (here and in few other places in this PR).